src/style.css

```diff
@@ -0,0 +1,23 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.first-block,
+.second-block,
+.third-block {
+  width: 100%;
```

Comment:
Block elements has 100% of the width of the parent container. Set width of 100% is redundant